test/format/wts.c

```diff
@@ -680,17 +686,16 @@ create_object(TABLE *table, void *arg)
 }
 
 /*
- * disagg_conn_init --
- *     For disaggregated storage, do some extra initialization of a connection.
+ * precise_checkpoint_init --
+ *     If precise checkpoint is enabled, do some extra initialization of a connection.
  */
 static void
-disagg_conn_init(WT_CONNECTION *conn)
+precise_checkpoint_init(WT_CONNECTION *conn)
```

Comment:
just confirming, if disagg is enabled without precise checkpoint this could be a problem?